Examples/test-suite/li_std_list.i

```diff
@@ -38,13 +38,23 @@ struct Struct {
   double num;
   Struct() : num(0.0) {}
   Struct(double d) : num(d) {}
-//  bool operator==(const Struct &other) { return (num == other.num); }
+  bool operator==(const Struct &other) { return (num == other.num); }
```

Comment:
Please revert this change. The `operator==` must remain commented out and the appropriate changes made to deal with it missing. This was mentioned in #989 and is because most classes do not have an `operator==` and std::list does not require the container type to have it.

Reply:
I've put in the required changes in 78051fdd33b5dc5fca0ec5787821ce5f9030af93